templates/usr/local/lib/owncloud-enforce-permissions.j2

```diff
@@ -5,11 +5,10 @@
 
 # {{ ansible_managed }}
 
-[[ "$UID" != 0 ]] && { echo "You must be root"; exit 1; }
-
```

Comment:
Since script is only able to run as root due to permissions I thought there is no reason to keep it.